types/auth0/index.d.ts

```diff
@@ -2171,4 +2172,7 @@ export class UsersManager<A = AppMetadata, U = UserMetadata> {
 
     impersonate(userId: string, settings: ImpersonateSettingOptions): Promise<any>;
     impersonate(userId: string, settings: ImpersonateSettingOptions, cb: (err: Error, data: any) => void): void;
+
+    getUserOrganizations(params: ObjectWithId): Promise<Organization[]>;
+    getUserOrganizations(params: ObjectWithId, cb: (err: Error, orgs: Organization[]) => void): void;
```

Comment:
```suggestion
    getUserOrganizations(data: ObjectWithId): Promise<Organization[]>;
    getUserOrganizations(data: ObjectWithId, cb: (err: Error, orgs: Organization[]) => void): void;
```
stick to the docs, it's a singular param

Reply:
oops, fixed